src/main/java/chess/domain/TeamColor.java

```diff
@@ -10,6 +10,13 @@ public TeamColor reverse() {
         return WHITE;
     }
 
+    public static TeamColor teamColor(String teamColor) {
+        if (teamColor.equalsIgnoreCase("WHITE")) {
```

Comment:
동치 비교 시에는 NPE 방지를 위해 항상 더 확실한 쪽을 기준으로 비교하는 것이 좋습니다.
```suggestion
        if ("WHITE".equalsIgnoreCase(teamColor)) {
```